src/command.ts

```diff
@@ -14,16 +14,19 @@ export abstract class CliCommand extends Command {
             `Authentication failure. Please obtain a new deployment key at https://app.subsquid.io and follow the instructions`,
           );
         case 400:
+          if (body.invalidFields) {
+            const messages = body.invalidFields.map(function (obj: any) {
+              return obj.message;
+            });
+            throw new CLIError(`Validation error. ${messages.join(' ')}`);
```

Comment:
why not  use `this.error` and return? 

And lets have each error on separete line 
```
this.error(`Validation error.\n\t ${messages.join("\n\t"))
```

Reply:
Sure, good idea.